dts/bindings/mbox/ti,omap-mailbox.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2024 Texas Instruments Incorporated.
+# SPDX-License-Identifier: Apache-2.0
+
+description: TI OMAP MAILBOX
+
+compatible: "ti,omap-mailbox"
+
+include: base.yaml
+
+properties:
+  reg:
+    required: true
+
+  interrupts:
+    required: true
+
+  usr-id:
+    type: int
+    required: true
+    description: User ID for processor
+
+mbox-cells:
+  - channel
```

Comment:
This can be removed if you include `mailbox-controller.yaml`

Reply:
The default value is not mentioned in the `mailbox-controller.yaml` and thus it defaults to 0 instead of 1, please let know if we should set the default as 1 and then remove the redundant `mbox-cells` entry in most of the bindings.